packages/wallet-ui/package.json

```diff
@@ -27,7 +26,6 @@
     "react-qr-code": "^2.0.7",
     "react-redux": "^8.0.1",
     "redux-persist": "^6.0.0",
-    "semver": "^7.3.7",
```

Comment:
semver is using in useStarkNetSnap.ts -> oldVersionDetected() , should not removed